docs/source/api/xskillscore.Contingency.accuracy.rst

```diff
@@ -1,4 +1,4 @@
-xskillscore.Contingency.accuracy
+﻿xskillscore.Contingency.accuracy
```

Comment:
When running the new api command, you only need to git add the newly generated file.

Reply:
I usually fix this by deleting everything in that folder, then re-running `sphinx-autogen` and adding it to git with `pre-commit` turned on. It'll recognize nothing has changed here and will drop these changes.